tensor2tensor/data_generators/translate_mnzh_bpe32k.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Created by Jack on 04/02/2018
+
```

Comment:
bpe in name suggests and external BPE dictionary will be downloaded, but this is not the case here